AudioKit/iOS/AudioKit/AudioKit.playground/contents.xcplayground

```diff
@@ -109,11 +107,10 @@
         <page name='Tracking Frequency'/>
         <page name='Tremolo'/>
         <page name='Triangular Wave Oscillator'/>
+        <page name='Untitled Pagebar Organ'/>
         <page name='Using Functions Part 2'/>
         <page name='Using Functions'/>
         <page name='Variable Delay Operation'/>
         <page name='Variable Delay'/>
-        <page name='Drawbar Organ'/>
-        <page name='AKTapeRecorder2'/>
```

Comment:
I don't understand why Xcode is so weird about the handling of items in this list.